lib/gvariant.nix

```diff
@@ -71,7 +71,7 @@ rec {
     else if isGVariant v then
       v
     else
-      throw "The GVariant type of ${v} can't be inferred.";
+      throw "The GVariant type of “${builtins.typeOf v}” can't be inferred.";
```

Comment:
Is it possible to include the *value* of `v` in this error message? That'd make it much more useful.

Reply:
I did it for integers. I am not aware of a nice way for general value.